dev/MRTCore/mrt/mrm/mrmmin/mrmmin.vcxproj

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\ProjectReunion.Build.Cpp.props" />
```

Comment:
Is there an msbuild property that represents "the solution root" so you could say `$(that)\dev\ProjectReunion.Build.Cpp.props` ?

Reply:
`$(SolutionDir)`

[Common macros for MSBuild commands and properties](https://docs.microsoft.com/cpp/build/reference/common-macros-for-build-commands-and-properties?view=msvc-160) is your friend :-)

Reply:
You don't typically want to use solution relative properties because solutions can be created externally and owned by the user. Your projects may, for example, be added to a user's grand build solution that lives outside expected paths and now these paths will break.